airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -75,10 +76,12 @@ def __init__(
         livy_conn_id: str = default_conn_name,
         extra_options: Optional[Dict[str, Any]] = None,
         extra_headers: Optional[Dict[str, Any]] = None,
+        auth_type: Optional[Any] = None
     ) -> None:
         super().__init__(http_conn_id=livy_conn_id)
         self.extra_headers = extra_headers or {}
         self.extra_options = extra_options or {}
+        self.auth_type = auth_type or self.auth_type
```

Comment:
What does `or self.auth_type` do?

Reply:
The inherited `HttpHook` sets a default value to `self.auth_type` which should be used when the argument `auth_type==None`.